webpack.dev.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+/* eslint-disable */
+const { merge } = require('webpack-merge');
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Not Addressed